### DISCUSSION TITLE
Disagreement in contamination estimations with/without raw data

### DISCUSSION THREAD

**Discussion by dlee598** [campbio/celda#392] [Q&A]
Hi team at Celda, 

I am processing some scRNA data through decontX as part of my QC pipeline and have a quick query about the inclusion of raw data. 

I have run the same data through decontX with and without specifying the raw data and get drastically different contamination estimates. 

Attached is an image of the same cells being compared against each other regarding their contamination estimates (x-axis is with the raw data included and y-axis is without raw data)  

[with_without_raw_decontX.pdf](https://github.com/campbio/celda/files/11250761/with_without_raw_decontX.pdf)

As you can see, this figure makes it tricky to define a contamination cut-off from which to retain my cells i.e. cells satisfying a 0.3 or lower threshold (without raw considered) may have a high contamination estimate when the raw is considered. 

This is a HTO-dataset where there are 4 samples contained within; could this have play a role into the effect I am seeing here? If so, would you recommend me demultiplexing the 4 samples first (most likely at the fastq level so that I can re-run through cell ranger and generate filtered and raw directories) and running decontX on each one separately?

**Comment by joshua-d-campbell** [comment]
Hi @dlee598, thanks for trying out or tool! Here are a few questions and thoughts:

1) Did you do any sorting (FACS or bead based) on your samples before loading into the 10X. If cell populations were excluded, they may not be present in the filtered matrix but their ambient material may still be present in the raw matrix droplets. So that could be a reason why you are seeing discrepancies.

2) It is also a bit strange that you don't get any cells with zero or low contamination by either method. Have you made some UMAPs/tSNEs and done some basic clustering to see if you still get all of your expected cell types? Note that you can supply your own cell type labels to decontX if the heuristic decontX clustering method does not capture all the cell types you want. 

3) Are you trying to decontaminate the HTO/cite-seq data or the RNA data? 

4) You should probably do the demultiplexing after alignment and running CellRanger so UMIs can be appropriately handled. Each cell should have a relatively higher level of an HTO ADT. Seurat has a good tutorial for demultiplexing HTOs. 

Lastly, I am going to move this to a Discussion thread as I am not sure it is an issue with the code itself.
